### PR TITLE
feat(migrate): add generate after db push

### PR DESCRIPTION
**Pull request by Jolg42**

### CODE REVIEW

**Comment by Jolg42** [comment]
Failing test is a flaky MySQL error, we can ignore here

File: src/packages/migrate/src/Migrate.ts [hunk 1]
```diff
@@ -268,6 +266,62 @@ export class Migrate {
...
+              version ? ` (version: ${version})` : ''
+            }${toStr} in ${formatms(after - before)}`,
+          )
+          generator.stop()
+        } catch (err) {
+          message.push(`${err.message}`)
+          generator.stop()
+        }
+      }
+    } catch (errGetGenerators) {
```
Comment (timsuchanek) [inline src/packages/migrate/src/Migrate.ts:318]: Why do you `catch` and then directly `throw` it again?

File: src/packages/migrate/src/Migrate.ts [hunk 2]
```diff
@@ -97,11 +97,10 @@ export class Migrate {
   // tslint:disable
   public watchUp = simpleDebounce(
```
Comment (timsuchanek) [inline src/packages/migrate/src/Migrate.ts:98]: We should probably remove this. But can also be done later.

File: src/packages/migrate/src/Migrate.ts (hunk 1 above)
Comment (Jolg42) [reply src/packages/migrate/src/Migrate.ts:318]: I guess because it's a copy paste from another place where we where doing more than throwing 😅

Comment (timsuchanek) [reply src/packages/migrate/src/Migrate.ts:318]: Ok, you can also merge and clean up after

Comment (Jolg42) [reply src/packages/migrate/src/Migrate.ts:318]: Sounds good to me!

**Pull request merged**